src/component/project/Project/ProjectHealth/ReportTable/ReportStatusCell/formatStatus.ts

```diff
@@ -1,5 +1,5 @@
 import { IFeatureToggleListItem } from 'interfaces/featureToggle';
-import { getDiffInDays, expired } from 'component/Reporting/utils';
+import { getDiffInDays, expired } from '../utils';
```

Comment:
Do we prefer a relative path in this case?

Reply:
This `utils` is related only to reporting, so I thing it's OK